packages/react/src/components/TextArea/TextArea.tsx

```diff
@@ -323,11 +323,17 @@ const TextArea = React.forwardRef((props: TextAreaProps, forwardRef) => {
 
   const formItemClasses = classNames(`${prefix}--form-item`, className);
 
-  const textAreaWrapperClasses = classNames(`${prefix}--text-area__wrapper`, {
-    [`${prefix}--text-area__wrapper--readonly`]: other.readOnly,
-    [`${prefix}--text-area__wrapper--warn`]: warn,
-    [`${prefix}--text-area__wrapper--slug`]: slug,
-  });
+  const textAreaWrapperClasses = classNames(
+    `${prefix}--text-area__wrapper`,
+    {
+      [`${prefix}--text-area__wrapper--cols`]: other.cols,
+    },
+    {
+      [`${prefix}--text-area__wrapper--readonly`]: other.readOnly,
+      [`${prefix}--text-area__wrapper--warn`]: warn,
+      [`${prefix}--text-area__wrapper--slug`]: slug,
+    }
+  );
```

Comment:
```suggestion
  const textAreaWrapperClasses = classNames(`${prefix}--text-area__wrapper`, {
    [`${prefix}--text-area__wrapper--cols`]: other.cols,
    [`${prefix}--text-area__wrapper--readonly`]: other.readOnly,
    [`${prefix}--text-area__wrapper--warn`]: warn,
    [`${prefix}--text-area__wrapper--slug`]: slug,
  });
```

minor formatting fix